### PR TITLE
dissolve subchapter added

### PR DESCRIPTION
**Pull request by KimboGilda**
@ratnanil I added the dissolve subchapter. Please check if anything needs to be adapted - or changed. In the menatime I continue with the other ones

### CODE REVIEW

**Review by ratnanil — APPROVED** [review]

**Comment by ratnanil** [comment]
@KimboGilda, can you resolve the conflicts before i merge the pull request?

**Comment by KimboGilda** [comment]
> 
> 
> @KimboGilda, can you resolve the conflicts before i merge the pull request?
@ratnanil  I think now you can merge it

**Comment by KimboGilda** [comment]
I replaced the datasets using the data("nameOfDataset") approach. Have a look @ratnanil , propose any changes and then we can merge it

File: .Rhistory [hunk 1]
```diff
@@ -1,57 +1,3 @@
-ggplot(zurich) + geom_sf()
```
Comment (ratnanil) [inline .Rhistory]: remove your .Rhistory from your cached files (see #65)!

Comment (ratnanil) [reply .Rhistory]: and also, really try out the RStudio-Git-GUI, you are going to love it. I'm really surpised that it took me so long to realize this. Just overwrite your `id_rsa` I dont think much will happen.. you'll love it ❤️

**Comment by KimboGilda** [comment]
I am sure I will. Unfortunately I could not find a way to replace the
default ssh key. I commited also everything and not a single file, because
I changed the datasets (arc2r) in many files.

File: 03_Data_Management/Raster_Processing_toolset.Rmd [hunk 2]
```diff
@@ -11,6 +11,16 @@
...
 
+For this chapter, you will need the following R Packages:
+
+```{r, echo = FALSE}
+library(arc2r)
+library(raster)
+library(stars)
+library(sf)
+library(dplyr)
+library(stars)
```
Comment (ratnanil) [inline 03_Data_Management/Raster_Processing_toolset.Rmd:22]: stars is listed twice

File: 03_Spatial_Analyst/Spatial_Analyst.Rmd [hunk 3]
```diff
@@ -134,10 +137,9 @@ Let's start by computing the _slope_ of the **swissAlti3d** raster dataset.
 ```{r}
 # Importing a raster dataset
 # swissALTI3D2019.tif -- Source: https://geovite.ethz.ch
```
Comment (ratnanil) [inline 03_Spatial_Analyst/Spatial_Analyst.Rmd:139]: if the source is from geovite, are you sure we are allowed to distribute it in this manner?

File: 03_Spatial_Statistics/Spatial_Statistics.Rmd [hunk 4]
```diff
@@ -49,9 +56,9 @@ morans_i <- function(sf_object,col) {
 --> 
 
 ```{r}
-zweitwohung <- read_sf("sample_data/other/zweitwohnung_gemeinden.gpkg")
+data("zweitwohnung_gemeinden")
 
-zweitwohung <- st_set_crs(zweitwohung,2056)
+zweitwohung <- st_set_crs(zweitwohnung_gemeinden,2056)
```
Comment (ratnanil) [inline 03_Spatial_Statistics/Spatial_Statistics.Rmd:61]: there is a typo zweitwoh**n**ung

File: 03_Spatial_Analyst/Spatial_Analyst.Rmd (hunk 3 above)
Comment (KimboGilda) [reply 03_Spatial_Analyst/Spatial_Analyst.Rmd:139]: I do not know per se, but the same dataset is also available in geodata2use. We could also have this as source

File: 03_Data_Management/Raster_Processing_toolset.Rmd (hunk 2 above)
Comment (KimboGilda) [reply 03_Data_Management/Raster_Processing_toolset.Rmd:22]: fixed

File: 03_Spatial_Analyst/Spatial_Analyst.Rmd (hunk 3 above)
Comment (ratnanil) [reply 03_Spatial_Analyst/Spatial_Analyst.Rmd:139]: geovite and geodata2use have restrict the distribution of data. Normally, we would need to replace this dataset at one point, but swissalti3d is going to be freely available as of 1.3.2021 it think

**Pull request merged**